build.gradle.kts

```diff
@@ -85,11 +86,15 @@ idea {
     project {
         settings {
             runConfigurations {
+                // Disable "condensed" multi-line diffs when running tests from the IDE (for both Gradle and JUnit test
+                // runners) to more easily accept actual results as expected results.
                 defaults<Gradle> {
-                    // Disable "condensed" multi-line diffs when running tests from the IDE to more easily accept actual
-                    // results as expected results.
                     jvmArgs = "-Dkotest.assertions.multi-line-diff=simple"
                 }
+
+                defaults<JUnit> {
+                    vmParameters = "-Dkotest.assertions.multi-line-diff=simple"
+                }
```

Comment:
I just tested this and the param still doesn't get added for me. The run configurations created by the Kotest plugin are in a separate section called "Kotest", is it possible to add this here as well?

Reply:
Unfortunately not, as the "Kotest" extension is not known to the idea-ext plugin / the core of IDEA itself. Are you sure you're running on the latest version of the Kotest plugin?

Reply:
I'm not, I'll upgrade later to check if that's related.